src/book/helpers/accountType.ts

```diff
@@ -1,10 +1,17 @@
 import type { Account } from '@/book/entities';
 
-export const ASSET_ACCOUNTS = ['ASSET', 'BANK', 'CASH', 'STOCK', 'MUTUAL', 'RECEIVABLE'];
+export const ASSET_ACCOUNTS = [
+  'ASSET',
+  'BANK',
+  'CASH',
+  'FIXED',
```

Comment:
FIXED assets are assets that can't be sold easily like vehicles, land, buildings, etc.